packages/editor-ui/src/components/CredentialsInput.vue

```diff
@@ -218,6 +234,37 @@ export default mixins(
 
 			return this.credentialDataTemp;
 		},
+		isCredentialTestable (): boolean {
+			if (['oAuth1Api', 'oAuth2Api'].includes(this.credentialTypeData.name)) {
+				return false;
+			}
```

Comment:
Some node credentials names are [missing the `Api` suffix](https://github.com/n8n-io/n8n/blob/d17ab8e9e571bae404b08a8d2a5468d8bb994c13/packages/nodes-base/credentials/GoogleAnalyticsOAuth2Api.credentials.ts#L13).

Reply:
Thanks for pointing it out! Not a problem here, actually.

In this case, we're not looking specifically for the suffix but rather the literal class name. This is because on the next `if` block we are looking for all superclasses.

If it's an OAuth mechanism, it should extend the base `oAuth1Api` or `oAuth2Api`.